programme/euler-number/EulersNumber.java

```diff
@@ -5,7 +5,13 @@
 public class EulersNumber {
 
     public static double eulersNumber(int n) {
-
+        
+        /* this can also be done using the Java Math library:
+        
+           e = Math.exp(n);
```

Comment:
But both will give different output 🤔

Reply:
I should've done more testing. Does the current solution give the proper output? 
And it looks like I was wrong... I think: ["public static double exp(double a) Returns Euler's number e raised to the power of a double value."](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/lang/Math.html#exp(double))

Reply:
`Math.exp(n);` will give solution for `e^x`, but we want

```
1/0! + 1/1! + ... + 1/n!
```

So, it will not work in this case

> Does the current solution give the proper output?

Yes